### PR TITLE
feat(multistream): Experimental multistream

### PR DESCRIPTION
**Pull request by Entrivax**
Modification done by this PR:
- Return the channel with each VOD in the playlist request
- Return multistream information with the playlist when requested
- Save a delay with each VOD in a playlist for multistream synchronization

This PR goes in pair with https://github.com/Zibbp/ganymede-frontend/pull/25 that adds the multistream page to the frontend.

### CODE REVIEW

**Comment by Zibbp** [comment]
Don't worry about the failing tests, those are not in a good state right now and require ENV vars from this repository.

**Comment by Entrivax** [comment]
Okay! Let me know if there is more changes you want to see on this PR (and the front-end one)!

File: internal/transport/http/handler.go [hunk 1]
```diff
@@ -246,6 +246,7 @@ func groupV1Routes(e *echo.Group, h *Handler) {
 	playlistGroup.POST("/:id", h.AddVodToPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
 	playlistGroup.DELETE("/:id/vod", h.DeleteVodFromPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
 	playlistGroup.DELETE("/:id", h.DeletePlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
+	playlistGroup.PUT("/:id/delay", h.SetVodDelayOnPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
```
Comment (Zibbp) [inline internal/transport/http/handler.go]:
```suggestion
	playlistGroup.PUT("/:id/multistream/delay", h.SetVodDelayOnPlaylist, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
```
Adding `multistream` to the URL makes it move obvious what the delay is for. You will also need to update the frontend API call to this.

Comment (Zibbp) [inline internal/transport/http/handler.go]:
```suggestion
	playlistGroup.PUT("/:id/delay", h.SetVodDelayOnPlaylistMultistream, auth.GuardMiddleware, auth.GetUserMiddleware, auth.UserRoleMiddleware(utils.EditorRole))
```
Adding `Multistream` to the function just so it's more clear what the function is. The function name will need to be updated in the other places as well.

**Review by Zibbp — COMMENTED** [review]
Looks good! I added two suggestions about better naming clarity.
Thanks for the PR!

File: internal/transport/http/handler.go (hunk 1 above)
Comment (Entrivax) [reply internal/transport/http/handler.go]: Updated here and on the frontend!

Comment (Entrivax) [reply internal/transport/http/handler.go]: Updated!

**Review by Zibbp — APPROVED** [review]

**Pull request merged**